LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -258,6 +303,42 @@ class LockboxXCUITests: BaseTestCase {
         waitforExistence(app.tables.cells.staticTexts["iosmztest@gmail.com"])
     }
 
+    func test8SearchOptions() {
+        navigator.goto(Screen.LockboxMainPage)
+        let searchTextField = app.cells.textFields["filterEntries.textField"]
+        searchTextField.tap()
+        searchTextField.typeText("a")
+        // There should be two matches
+        let twoMatches = app.tables.cells.count-1
+        XCTAssertEqual(twoMatches, 2)
+
+        // There should be one match
+        searchTextField.typeText("cc")
+        let oneMatches = app.tables.cells.count-1
+        XCTAssertEqual(oneMatches, 1)
+
+        // There should not be any matches
+        searchTextField.typeText("x")
+        sleep(2)
```

Comment:
Please add a comment on why `sleep` is necessary. Or try and avoid adding `sleep`s.